packages/autocomplete-core/src/onKeyDown.ts

```diff
@@ -105,7 +105,7 @@ export function onKeyDown<TItem extends BaseItem>({
     // pending and could reopen the panel once they resolve, because that would
     // result in an unsolicited UI behavior.
     store.pendingRequests.cancelAll();
-  } else if (event.key === 'Enter') {
+  } else if (event.key === 'Enter' && !event.isComposing) {
```

Comment:
https://github.com/github/combobox-nav/blob/main/src/index.ts#L101 we may need to ignore everything when composing